docs/source/tutorials/image.ipynb

```diff
@@ -268,21 +605,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "num_crossval_folds = 3  # for efficiency; values like 5 or 10 will generally work better\n",
-    "pred_probs = cross_val_predict(\n",
-    "    model_skorch,\n",
-    "    X,\n",
-    "    labels,\n",
-    "    cv=num_crossval_folds,\n",
-    "    method=\"predict_proba\",\n",
-    ")"
+    "indices = np.hstack(test_id_list)\n",
+    "dataset = dataset.select(indices)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "An additional benefit of cross-validation is that it facilitates more reliable evaluation of our model than a single training/validation split."
+    "## 7. Use cleanlab to find issues\n",
+    "\n",
+    "Based on the given labels, out-of-sample predicted probabilities and feature embeddings, cleanlab can quickly help us detect potential issues in our dataset. \n",
+    "\n",
+    "Here, we use cleanlab's `Datalab` class to find potential issues in our data. `Datalab` has several ways of loading the data. For our current scenario, we are employing the huggingface dataset. `Datalab` takes in two optional arguments, `label_name`, which corresponds to the label column's name, and `image_key`, corresponding to the image column's name. When you provide these optional arguments, `Datalab` will audit your dataset for more types of issues than it would by default."
```

Comment:
```suggestion
    "Here we use cleanlab's `Datalab` class to find issues in our data. `Datalab` supports several data formats, in this tutorial we have a Hugging Face Dataset. `Datalab` takes in two optional dataset arguments:  `label_name`, which corresponds to the column containing labels (if your dataset is labeled), and `image_key`, corresponding to the  name of a key in your vision dataset to access the raw images. When you provide these optional arguments, `Datalab` will audit your dataset for more types of issues than it would by default."
```